src/main.cpp

```diff
@@ -981,6 +981,36 @@ bool ContextualCheckTransaction(
             // https://zips.z.cash/zip-0213#specification
             uint256 ovk;
             for (const OutputDescription &output : tx.vShieldedOutput) {
+                bool zip_212_enabled;
+                libzcash::SaplingPaymentAddress zaddr;
+                CAmount value;
+
+            // 5.4.2 EoS height is 2121024.
+            if (nHeight >= 2121200) {
```

Comment:
The negative indentation here is intentional, both to reduce the overall diff, and to reduce the diff of the future PR that removes the old code.